src/main/java/org/zalando/nakadi/repository/db/EventTypeDbRepository.java

```diff
@@ -86,7 +92,7 @@ public EventType mapRow(final ResultSet rs, final int rowNum) throws SQLExceptio
     @Override
     public List<EventType> list() {
```

Comment:
I think you should also adjust saving of event-type. Because with current implementation it will not be possible for user to delete event-type and then create another one with the same name (method with throw DuplicatedEventTypeNameException)